src/librustc/middle/cstore.rs

```diff
@@ -122,7 +122,6 @@ pub struct ExternCrate {
 /// can be accessed.
 pub trait CrateStore<'tcx> {
     // item info
-    fn describe_def(&self, def: DefId) -> Option<Def>;
```

Comment:
Right, I think the correct thing here is to split the `DefId` if we need them separate.